frontend/src/services/eventHandlers/gameHandler.js

```diff
@@ -48,15 +54,15 @@ export default class GameHandler {
                 const selectedWord = store.getters["scribble/getWordSelected"];
                 const state = new Drawing(
                     selectedWord,
-                    payload.playTimeSend.duration
+                    this._convertToSeconds(payload.playTimeSend.duration)
                 );
                 store.commit(this.setGameStateKey, state);
             } else if (
                 currentState.state === WaitingForPlayerToChooseWord.STATE
             ) {
                 const state = new Guessing(
                     payload.playTimeSend.hint,
-                    payload.playTimeSend.duration
+                    this._convertToSeconds(payload.playTimeSend.duration)
```

Comment:
When calling here, it's not clear what the parameter should be. I think it should be renamed to `convertNanoSecsToSecs